xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1168 @@
+
+              <entry>Make sure the project you declared in the
+              .obs/workflows.yml really exists in OBS.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>"Package not found"</emphasis></entry>
+
+              <entry>Make sure the package you declared in the
+              .obs/workflows.yml really exists in OBS.</entry>
+            </row>
+
+            <row>
+              <entry>No build result updates are displayed in your
+              PR/MR</entry>
+
+              <entry>Make sure there are repositories defined on your source
+              project. Another reason can be that the build gets stuck in a
+              statuses like "unresolvable" or "broken"; in that case, ask for
+              <link
+              xlink:href="https://openbuildservice.org/support">help</link>.</entry>
+            </row>
+
+            <row>
+              <entry>The project in OBS doesn't get updated with the latest
+              changes in the SCM.</entry>
+
+              <entry>For certain steps you need to set up a
+              <emphasis>_service</emphasis> file. Follow the <link
+              xlink:href="https://github.com/openSUSE/obs-service-tar_scm#user-documentation">obs-service-tar_scm</link>
+              documentation.</entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.equivalence_table">
+      <title>Equivalence Table</title>
+
+      <table>
+        <title>Equivalence table</title>
+
+        <tgroup align="left" cols="2">
+          <thead>
+            <row>
+              <entry>GitHub</entry>
+
+              <entry>GitLab</entry>
+            </row>
+          </thead>
+
+          <tbody>
+            <row>
+              <entry>Repository</entry>
+
+              <entry>Project</entry>
+            </row>
+
+            <row>
+              <entry>Pull request</entry>
+
+              <entry>Merge request</entry>
+            </row>
+
+            <row>
+              <entry>PR</entry>
+
+              <entry>MR</entry>
+            </row>
+
+            <row>
+              <entry>Push</entry>
+
+              <entry>Push Hook</entry>
+            </row>
+
+            <row>
+              <entry>Pull requests (in webhook configuration)</entry>
+
+              <entry>Merge request events (in webhook configuration)</entry>
+            </row>
+
+            <row>
+              <entry>Pushes (in webhook configuration)</entry>
+
+              <entry>Push events (in webhook configuration)</entry>
+            </row>
+
+            <row>
+              <entry>repo (required personal access token scope)</entry>
+
+              <entry>api (required personal access token scope)</entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+    </sect2>
+  </sect1>
+
+  <sect1 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases">
```

Comment:
Maybe you want to split both `<sect1>` into two files?

Reply:
I can do it, but we agreed on not doing it now.